src/main/java/Tasks/Deadlines.java

```diff
@@ -0,0 +1,29 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+public class Deadlines extends Task {
+    private LocalDateTime deadlineInfo;
+    private static final DateTimeFormatter PATTERN_DISPLAY = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+    private static final DateTimeFormatter PATTERN_READ = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+
+    public Deadlines(String name, String details) {
+        super(name);
+        this.deadlineInfo = LocalDateTime.parse(details, PATTERN_READ);
```

Comment:
Here the reference to the field `deadlineInfo` is not ambiguous, can possibly omit `this` keyword